process/css-styleguide/README.md

```diff
@@ -0,0 +1,168 @@
+# Meilisearch demos style guide
+
+Meilisearch Demos style guide contains a set of common UI elements along with their CSS for any developer to choose from to maintain consistent styling across the demos created for Meilisearch. If you are interested in creating a demo, you can refer to the common UI components and use those in your project.
+
+## About the style guides
+
+The style guide consists of a number of common UI components such as buttons, headings, search inputs that can be included while creating the UI of a project. It is based on the [Meilisearch Design System](https://www.figma.com/file/XXmUvsZzfZBKt1i2v12wf4/Design-System?node-id=1504%3A282).
+
+For creating beautiful in-brand demos, we encourage you to consult the [Meilisearch Guide to Design a Demo](https://www.figma.com/file/YUqMlOmApYd0D2aitPyDce/Guide-to-design-a-demo?node-id=2%3A8).
+
+## Components included
+
+- Button
+- Input
+- Logos
+- Modal
+- Search Input
+- Tag
+- Typography: headings and body text
+
+## Usage
+
+### Button
+
+1. The base class for styling a button is `.btn`.
+2. Change the color styling of the button by using the following classes:
+- `.btn-dodger-blue` 
+- `.btn-hot-pink`
+- `.btn-lila` 
+- `.btn-turquoise`
+- `.btn-watermelon`
+- `.btn-peppermint`
+- `.btn-candlelight`
+
+```html
+<button class="btn btn-dodger-blue" type="button">Button</button>
+```
+
+3. You can create a secondary button by adding the `btn-secondary` class
```

Comment:
Sometimes you refer to the classes with a dot before the class name, and sometimes not. Can you standardize this? 🙏

Reply:
I hope I haven't missed any others!